hack/build/ci/generate-release-notes.sh

```diff
@@ -44,10 +39,10 @@ For information on how to install the [latest dynatrace-operator](https://github
 
 #### Helm (recommended)
 \`\`\`sh
-helm upgrade dynatrace-operator oci://public.ecr.aws/dynatrace/dynatrace-operator \
-  --version ${tag_without_leading_v} \
-  --create-namespace --namespace dynatrace \
-  --install \
+helm upgrade dynatrace-operator oci://public.ecr.aws/dynatrace/dynatrace-operator \\
+  --version ${tag_without_leading_v} \\
```

Comment:
when i run the script at this place I get this output:


```
helm upgrade dynatrace-operator oci://public.ecr.aws/dynatrace/dynatrace-operator \
  --version tag> \
  --create-namespace --namespace dynatrace \
  --install \
  --atomic
```

```
  --version tag> \
```
this line looks weird. Can you confirm or does this only happen on my machine?

Reply:
yes, that is kinda ugly, but happens for me aswell, but it happens irrelevant of my change

its because of `tag=${GITHUB_REF_NAME:-<tag>}`, where `<tag>` is just a place holder, it will always be `GITHUB_REF_NAME` when its actually run

(so I wouldn't worry about it 😅 )